### DISCUSSION TITLE
twitch gql api

### DISCUSSION THREAD

**Discussion by backink** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#479] [Q&A]
Hello
How do you know the json columns and valid operations of twitch gql api?
I couldn't find any doc online.
Some operations and info couldn't be done by formal twitch api.
Thanks

**Comment by Tkd-Alex** [comment]
Use a proxy like Charles or BurpSuite, or just inspect the network tab from Google Chrome Developer Tools
